src/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import {
+  Provider,
+} from 'react-redux';
+import {
+  createStore,
+} from 'redux';
+import App from './components/App';
+import * as serviceWorker from './serviceWorker';
+import combineReducers from './reducers/index';
+
+const initialState = {
+  categories: [{
+    name: 'Action',
+  },
+  {
+    name: 'Biography',
+  },
+  {
+    name: 'History',
+  },
+  {
+    name: 'Horror',
+  },
+  {
+    name: 'Kids',
+  },
+  {
+    name: 'Learning',
+  },
+  {
+    name: 'Sci-Fi',
+  },
+  ],
```

Comment:
I don't think that it is actually needed here. You just need an array of string that is responsible to render the options while creating a book or filtering specific categories book.